packages/wallets/src/evm/wallets/local-wallet.ts

```diff
@@ -380,85 +380,85 @@ export class LocalWallet extends AbstractClientWallet<
 
 type DecryptOptions =
   | {
-    decrypt?: (message: string, password: string) => Promise<string>;
-    password: string;
-  }
+      decrypt?: (message: string, password: string) => Promise<string>;
+      password: string;
+    }
```

Comment:
oof, do we have different settings here?

Reply:
sadge we must